src/pyclvm/instance/ls.py

```diff
@@ -30,20 +28,20 @@ def ls(**kwargs: str) -> None:
         None
 
     """
-    instances = Ec2InstanceMapping(get_session(kwargs))
+    instances = Ec2AllInstancesData(**kwargs)
```

Comment:
doesn't seem like kwargs is needed, might need to patch in Ec2AllInstancesData inside ec2map as well